arrows/core/tests/test_derive_metadata.cxx

```diff
@@ -60,7 +60,7 @@ kv::image_container_scptr
 make_image()
 {
   constexpr auto frame_height = size_t{ 720 };
-  constexpr auto frame_width = size_t{ 1080 };
```

Comment:
Also probably my fault.

Reply:
Heh, I was wondering about that; 3:2 isn't a bonkers aspect ratio though...